recipes-devtools/sbuild-chroot/sbuild-chroot-target-npm.bb

```diff
@@ -0,0 +1,6 @@
+DESCRIPTION = "Isar sbuild/schroot filesystem for target (npm variant)"
+
+require recipes-devtools/sbuild-chroot/sbuild-chroot-target.bb
+
+SBUILD_FLAVOR = "npm"
+SBUILD_CHROOT_PREINSTALL_EXTRA ?= "npm dpkg-dev"
```

Comment:
As discussed in #433, this will need more. I wonder why you didn't check that MR.

Reply:
I didn't see this pull-request. Will look later.

Reply:
Looked a bit at #433. It seems to be related just to switching to derived chroot (with "npm" preintalled), while the patchset I propose is mostly about complete removing of buildchroot from npm.bbclass.

Reply:
Sure, but that does not mean we should ignore what we discussed there.